client/components/select-dropdown/index.jsx

```diff
@@ -243,6 +245,7 @@ class SelectDropdown extends Component {
 					aria-owns={ 'select-submenu-' + this.state.instanceId }
 					aria-controls={ 'select-submenu-' + this.state.instanceId }
 					aria-expanded={ this.state.isOpen }
+					aria-disabled={ this.props.disabled }
```

Comment:
What's the effect of `aria-disabled` here? Do screen readers handle content-like tags OK?